internal/adapters/operation_lease/redis/redis.go

```diff
@@ -0,0 +1,66 @@
+// MIT License
+//
+// Copyright (c) 2021 TFG Co
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in all
+// copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+// SOFTWARE.
+
+package redis
+
+import (
+	"context"
+	"time"
+
+	"github.com/topfreegames/maestro/internal/core/entities/operation"
+
+	"github.com/go-redis/redis/v8"
+	"github.com/topfreegames/maestro/internal/core/ports"
+)
+
+var _ ports.OperationLeaseStorage = (*redisOperationLeaseStorage)(nil)
+
+// redisOperationLeaseStorage adapter of the OperationLeaseStorage port. It store store
+// the operations in lists to keep their creation/update order.
```

Comment:
It is not going to store the operation lease in a list, right? we have defined that we will use a sorted set according to RFC